freqtrade/optimize/optimize_reports.py

```diff
@@ -509,12 +516,22 @@ def generate_backtest_stats(btdata: Dict[str, DataFrame],
     :param max_date: Backtest end date
     :return: Dictionary containing results per strategy and a strategy summary.
     """
-    result: Dict[str, Any] = {'strategy': {}}
+    result: Dict[str, Any] = OrderedDict([
+        ('metadata', {}),
+        ('strategy', {}),
+        ('strategy_comparison', []),
+    ])
     market_change = calculate_market_change(btdata, 'close')
+    metadata = result['metadata']
```

Comment:
i honestly find this assignment slightly missleading, and dangerous to introduce bugs in the future.

I understand it's just a "convenience" to not have to use `result['metadata']['strategy'] =` below - but for someone not 100% aware that there will only be one metadata dict this might be confusing i think.